docker-compose.helsinki-benefit.yml

```diff
@@ -0,0 +1,55 @@
+version: '3.8'
+services:
+    postgres:
+        image: postgres:12
+        restart: on-failure
+        environment:
+            POSTGRES_USER: helsinkibenefit
+            POSTGRES_PASSWORD: helsinkibenefit
+            POSTGRES_DB: helsinkibenefit
+        ports:
+            - 5434:5432
+        volumes:
+          - pgdata:/var/lib/postgresql/data
+        container_name: helsinkibenefit-db
+
+    django:
+        build:
+            context: ./backend
+            target: development
+        env_file:
+            - .env.helsinki-benefit
+        environment:
+            DATABASE_URL: postgres://helsinkibenefit:helsinkibenefit@helsinkibenefit-db/helsinkibenefit
+            DATABASE_HOST: helsinkibenefit-db.helsinki
+        volumes:
+            - ./backend/helsinki-benefit:/app
```

Comment:
Kristian is adding backend/shared directory in https://github.com/City-of-Helsinki/yjdh/pull/38
Once that is there, how should we include it here?

Reply:
yeah, we should add the common directory to the app volume. But I think this can be done together in the common component integration later.